masterlist.yaml

```diff
@@ -4892,6 +4892,76 @@ plugins:
     clean:
       - crc: 0x4E4A0351 # v2.0.3
         util: 'SSEEdit v3.2.71 EXPERIMENTAL'
+  - name: 'Distinct Interiors.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/6130/' ]
+    group: *addonsGroup
+    after:
+      - 'EnhancedLightsandFX.esp'
+      - 'ELFX - NoPlayerHomes.esp'
+      - 'Snazzy Furniture and Clutter Overhaul.esp'
+    inc:
+    # CRC for Modular - Core - Always Needed Version
+    # v1.81 B337055B,
```

Comment:
Core & Main modular plug-in share the name `Distinct Interiors.esp`. 
Core version receives more frequent updates than modular version.
So for now it made more sense to use that version for any checksum.